crates/sui-core/src/checkpoints/checkpoint_executor/mod.rs

```diff
@@ -149,7 +154,14 @@ impl CheckpointExecutor {
                     highest_executed = Some(checkpoint);
                 }
                 // Check for newly synced checkpoints from StateSync.
-                _ = self.mailbox.recv() => (),
+                Ok(checkpoint) = self.mailbox.recv() => {
+                    SystemTime::now().duration_since(checkpoint.summary.timestamp())
+                        .map(|latency|
+                            self.metrics.checkpoint_contents_age_ms.report(latency.as_millis() as u64)
+                        )
+                        .tap_err(|err| warn!("unable to compute checkpoint age: {}", err))
+                        .ok();
```

Comment:
can we add a log here with checkpoint height

Reply:
done